### PR TITLE
Adds Moffuchi's Pizzeria as a ruin to the surface of Icebox

### PR DESCRIPTION
**Pull request by Capsandi**
## About The Pull Request

Adds a 'new' ruin salvaged from #51867
The ghost roles are gone, and in there place lies signs of a crowded restaurant evacuating in the aftermath of a harrowing incident.
I also gutted the scrubbers net, pacman generator and drained the APC's power, reducing the hospitality of the venue.

## Why It's Good For The Game

it has been a year and a half-ish since the original pr and icebox still only has the ruins it shipped with (plus that etymologist ruin, very cool). No harm can be done by introducing another ruin to the pool.

image

## Changelog

:cl:
add: There's been a Moffuchi's Family Pizzeria on the surface of the icemoon but we only just noticed it
/:cl:

### CODE REVIEW

File: code/game/area/areas/ruins/icemoon.dm [hunk 1]
```diff
@@ -41,3 +41,13 @@
 	icon_state = "dk_yellow"
 	mood_bonus = 1
 	mood_message = "<span class='nicegreen'>This place seems strangely serene.</span>\n"
+
+/area/ruin/pizzeria
+	name = "\improper Moffuchis Pizzeria"
+	icon_state = "red"
+
+/area/ruin/pizzeria/kitchen
+	name = "\improper Moffuchis Kitchen"
```
Comment (Mothblocks) [inline code/game/area/areas/ruins/icemoon.dm:50]:
```suggestion
	name = "\improper Moffuchi's Pizzeria"
	icon_state = "red"

/area/ruin/pizzeria/kitchen
	name = "\improper Moffuchi's Kitchen"
```

**Comment by ATH1909** [comment]
is that the bite of '87??

**Comment by peptron1** [comment]
It has a moth plush, I like it.

**Comment by Bawhoppen** [comment]
if there's no ghost roles then :+1:

**Comment by Zytolg** [comment]
I have to point this out, weather or not it's to the detriment or benefit to this PR.

How is it, that when a Pizzaria is shot out of space, it somehow makes its way onto the ground? 
I have no clue how that's going to effect this PR, but it IS the 2nd Pizzaria PR I've seen this season. The ruin itself looks great, but I have a feeling...

**Comment by notfrying1pans** [comment]
what kinda milk do they use

**Pull request merged**